db/db_test.cc

```diff
@@ -1927,7 +1927,7 @@ namespace {
 
 static const int kColumnFamilies = 10;
 static const int kNumThreads = 10;
-static const int kTestSeconds = 10;
+static const int kTestSeconds = 1;
```

Comment:
Are you sure we want to reduce test time?

Reply:
Yes, in fact I think we should delete this test entirely as it should be covered by `db_stress`. But that's a longer-term thing as we need to gain more confidence in `db_stress` first.

Reply:
But db_stress is not a test but utility?

Reply:
That's true, though this test seems to be looking for problems in a brute force way, whereas our other tests cover particular features or edge cases. `db_stress` is dedicated to that purpose, and we run it continuously. The main reason, though, is travis keeps showing out-of-space errors so I want to get rid of the space-heavy unit tests :/.